tests/README.md

```diff
@@ -13,11 +13,12 @@ Note: Not all tests have been converted to tox and/or pytest yet.
 
 ## Running the tests
 
-Before running vdsm tests, activate the environment:
+Before running Vdsm tests, activate the virtual environment (which you need to
+create once, with `make venv`, as described in [doc/development.md](doc/development.md)):
 
     source ~/.venv/vdsm/bin/activate
 
-When done, you can deactivate the environment:
+When done with testing, you can deactivate the environment:
```

Comment:
WIth the test README I thought me discussed adding an example for pytest flags `--log-cli-level=DEBUG`, maybe also  `--cov-fail-under 0`.

But we can leave it for another PR, more specific for tests, so that we can move this forward.

Reply:
I added --no-cov as an example, I think that is simpler than setting the limit to 0.